lib/compileTemplate.ts

```diff
@@ -28,6 +28,7 @@ export interface TemplateCompileOptions {
 }
 
 export interface TemplateCompileResult {
+  ast: Object | void
```

Comment:
```suggestion
  ast: Object | undefined
```

Because `void` means absence of return type of a function.
(Same for README.md)

Reply:
Right 😅